test/jdk/javax/swing/JButton/bug4490179.java

```diff
@@ -49,13 +47,16 @@ public class bug4490179 {
     public static void main(String[] args) throws Exception {
         Robot robot = new Robot();
         robot.setAutoDelay(100);
+        robot.setAutoWaitForIdle(true);
         try {
             SwingUtilities.invokeAndWait(() -> {
                 frame = new JFrame("bug4490179");
                 button = new JButton("Button");
                 frame.getContentPane().add(button);
-                button.addActionListener(new ActionListener() {
-                    public void actionPerformed(ActionEvent e) {
+                button.addActionListener(e -> {
+                    if ((e.getModifiers() & InputEvent.BUTTON1_MASK)
```

Comment:
This check was lost for some reason during the open sourcing of the original test, so restored.

Reply:
Interesting. So since this was unconditionally setting "passed=false" how has the test been passing ?

Reply:
The action listener should be triggered when we release the Button1, and there was no such release in this test.
And we didn't wait long enough after that anyway, so it might pass even if it fails.